test/eviction/evictpod.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+// evictpod.go --pod <pod> --namespace <namespae> initiates a pod eviction
+// using the k8s eviction API.
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"path/filepath"
+
+	policy "k8s.io/api/policy/v1beta1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+// Borrowed from https://stackoverflow.com/questions/62803041/how-to-evict-or-delete-pods-from-kubernetes-using-golang-client
+func evictPod(client *kubernetes.Clientset, name, namespace string) error {
+	return client.PolicyV1beta1().Evictions(namespace).Evict(context.TODO(), &policy.Eviction{
```

Comment:
Is there a V1 version of this API (my quick look at godoc makes me think there is)? I think throughout most of our code base we no longer rely on beta APIs, which makes it so that we aren't at risk when APIs get removed in future k8s releases.

Reply:
Done